app/commands.go

```diff
@@ -77,7 +77,7 @@ type activated struct {
 	Validate  activatedValidateCmd `cmd:"" help:"Hermit validation." group:"global"`
 
 	Clean cleanCmd `cmd:"" help:"Clean hermit cache." group:"global"`
-	GC    gcCmd    `cmd:"" help:"Garbage collect unused Hermit packages and clean the download cache." group:"global"`
+	GC    gcCmd    `cmd:"" hidden:"" group:"global"`
```

Comment:
```suggestion
	GC    gcCmd    `cmd:"" hidden:"" group:"deprecated"`
```

Is there a way to that commands are deprecated?

Reply:
No there's no real concept of it in Kong. I just hide them and write a warning.